frontEnd/src/app/core/interfaces/auth/register-request.ts

```diff
@@ -0,0 +1,5 @@
+export interface IRegisterRequestInput {
+    name: string;
+    email: string;
+    password: string;
+}
```

Comment:
@KalindaJ-debug 
You can generalize `ILoginRequestInput ` and `IRegisterRequestInput `.  as follows. 
`
interface IAuthRequestInput {
    email: string;
    password: string;
}

interface IRegisterRequestInput extends IAuthRequestInput {
  name: string;
}
`

But your one is also good. you can keep your preferred one.